pulpcore/cli/core_cli.py

```diff
@@ -1,16 +1,17 @@
 import click
 
-from pulpcore.cli import main
+from pulpcore.cli.common import main, PulpContext
 from pulpcore.cli.core.artifact import artifact
 from pulpcore.cli.core.orphans import orphans
 from pulpcore.cli.core.task import task
 
 
 @main.command()
 @click.pass_context
-def status(ctx):
-    result = ctx.obj.call("status_read")
-    ctx.obj.output_result(result)
+def status(ctx: click.Context) -> None:
+    pulp_ctx: PulpContext = ctx.find_object(PulpContext)
+    result = pulp_ctx.call("status_read")
+    pulp_ctx.output_result(result)
```

Comment:
I'm curious why this changed as compared to say `orphans_delete()`?

Reply:
I didn't know about `find_object` but it looks like you can embed more than one object in the context that way.
I think, this is more explicit and forgot about `orphan_delete` it seems.